src/wasm32_bindgen.rs

```diff
@@ -64,9 +64,15 @@ fn getrandom_init() -> Result<RngSource, Error> {
         // `crypto.getRandomValues`, but if `crypto` isn't defined we assume
         // we're in an older web browser and the OS RNG isn't available.
```

Comment:
We should document the crypto vs mscrypto distinction here, so it's all in one place. Something like
```rust
        // If `self` is defined then we're in a browser somehow (main window
        // or web worker). We get `self.crypto` (called `msCrypto` on IE), so we
        // can call `crypto.getRandomValues`. If `crypto` isn't defined, we
        // assume we're in an older web browser and the OS RNG isn't available.
```